### PR TITLE
Update links to .NET SDK and resources

### PR DESCRIPTION
**Pull request by Roblinde**
Removed the unofficial notice from the .NET SDK and updated links to correct resources.

Should not be merged before https://github.com/contentful/contentful.net repo is reviewed and made public.

### CODE REVIEW

**Comment by ChrisChinchilla** [comment]
@Roblinde Reviewed, no major changes, but wont merge until I've moved the tutorials.

**Comment by ChrisChinchilla** [comment]
@Roblinde Can you take a look at this edit so far for me? I've cut a reasonable amount, but it needs a bit more. My main concern at the moment is that there is repetition between the 'Your first request' and 'Querying for content'.

We could either reduce that duplication or break it into separate articles.

File: docs/net/tutorials/using-net-cda-sdk.md [hunk 1]
```diff
@@ -4,17 +4,14 @@ page: ':docsUsingNetCdaSdk'
...
-
-This article goes into detail about how to get content using the [.Net CDA SDK](https://github.com/contentful/contentful.net).
+To make development easier for our users, we publish SDKs for various languages which make the task easier. This article details how to us the [.Net CDA SDK](https://github.com/contentful/contentful.net).
 
 ## Pre-requisites
 
-In this tutorial, we assume you understand the basic Contentful data model as described in the [developer center](/developers/docs/concepts/data-model/).
+This tutorial assumes you understand the basic Contentful data model as described in the [developer center](/developers/docs/concepts/data-model/).
 
 Contentful.Net is built on .net core and targets .Net Standard 1.6\. The SDK is cross platform and runs on Linux, macOS or Windows.
```
Comment (Roblinde) [inline docs/net/tutorials/using-net-cda-sdk.md]: Changed this to target .Net Standard 1.4 for better compatability.

File: docs/net/tutorials/using-net-cda-sdk.md [hunk 2]
```diff
@@ -13,7 +10,7 @@ To make development easier for our users, we publish SDKs for various languages
 
 This tutorial assumes you understand the basic Contentful data model as described in the [developer center](/developers/docs/concepts/data-model/).
 
-Contentful.Net is built on .net core and targets .Net Standard 1.6\. The SDK is cross platform and runs on Linux, macOS or Windows.
+Contentful.net is built on .net core and targets .Net Standard 1.6\. The SDK is cross platform and runs on Linux, macOS and Windows.
```
Comment (Roblinde) [inline docs/net/tutorials/using-net-cda-sdk.md]: Changed this to target .Net Standard 1.4 for better compatability.

**Pull request merged**